process/block/processedMb/processedMiniBlocks.go

```diff
@@ -9,37 +9,37 @@ import (
 
 var log = logger.GetOrCreate("process/processedMb")
 
-// ProcessedMiniBlockInfo will keep the info about processed mini blocks
+// ProcessedMiniBlockInfo will keep the info about a processed mini block
 type ProcessedMiniBlockInfo struct {
 	FullyProcessed         bool
 	IndexOfLastTxProcessed int32
 }
 
-// MiniBlocksInfo will keep a list of miniblocks hashes as keys, with miniblocks info as value
-type MiniBlocksInfo map[string]*ProcessedMiniBlockInfo
+// miniBlocksInfo will keep a list of mini blocks hashes as keys, with mini blocks info as value
+type miniBlocksInfo map[string]*ProcessedMiniBlockInfo
 
-// ProcessedMiniBlockTracker is used to store all processed mini blocks hashes grouped by a metahash
-type ProcessedMiniBlockTracker struct {
-	processedMiniBlocks    map[string]MiniBlocksInfo
+// processedMiniBlocksTracker is used to store all processed mini blocks hashes grouped by a meta hash
+type processedMiniBlocksTracker struct {
+	processedMiniBlocks    map[string]miniBlocksInfo
 	mutProcessedMiniBlocks sync.RWMutex
 }
 
-// NewProcessedMiniBlocks will create a complex type of processedMb
-func NewProcessedMiniBlocks() *ProcessedMiniBlockTracker {
-	return &ProcessedMiniBlockTracker{
-		processedMiniBlocks: make(map[string]MiniBlocksInfo),
-	}
+// NewProcessedMiniBlocksTracker will create a processed mini blocks tracker object
+func NewProcessedMiniBlocksTracker() (*processedMiniBlocksTracker, error) {
```

Comment:
returning always nil for the error will result in a lot of dead code where we use this constructor. Please remove the error return param

Reply:
Done